eland/ml/ml_model.py

```diff
@@ -460,6 +441,3 @@ def _trained_model_config(self) -> Dict[str, Any]:
                 raise ValueError(f"Model with Model ID {self._model_id!r} wasn't found")
             self._trained_model_config_cache = resp["trained_model_configs"][0]
         return self._trained_model_config_cache
-
-
-ImportedMLModel = deprecated_api("MLModel.import_model()")(MLModel.import_model)
```

Comment:
Let's leave this alias here for some time, it was only deprecated 3 months ago. However all other changes for `ImportedMLModel` are good to keep!